run.py

```diff
@@ -290,6 +297,12 @@ def setup_parser():
                       help="Regression testlist")
   parser.add_argument("--csr_yaml", type=str, default="",
                       help="CSR description file")
+  parser.add_argument("--pass_val", type=int, default=1,
+                      help="Value that privileged CSR test writes on success")
+  parser.add_argument("--fail_val", type=int, default=2,
```

Comment:
Use 0 for failure?

Reply:
Sure, can switch to zero. I initially wanted to avoid any false positive EOT checks since 0 is a common default value